### DISCUSSION TITLE
Issue with plugin ID change between beta 5 and beta 6 and request

### DISCUSSION THREAD

**Discussion by Henri75** [apohl79/audiogridder#772] [Help]
So far it seems that beta 6 with update Juce framework has broken plugin ID. It means that some (a lot, but not all) plugin wont be find anymore because their internal ID has changed.

Not sure there is an easy workaround for that, and it breaks totally compatibility with old project.
But I guess that the priority is stability and support of recent juce version. so I can live with that

**One simple workaround would be at least to load the AG chain and make it run even if a plugin is not found in the chain (see how cubase handle that when a plugin is missing, it bypass it and highlight the missing plugin which exclamation marks).**
Can you implement that ?

Currently it is the worst scenario : sandbox terminated if a single plugin is not found and the whole AG chain and insert is disconnected and stop sounds.

**Comment by apohl79** [comment]
The problem is, I can't confirm this (yet). All of my plugins work perfectly fine. So I need a server log of pre-beta6 and beta6 where the server performs a full plugin scan. That way - if it is an issue - I will see the plugins with different IDs and can focus on them. But so far I believe JUCE is fine.

**Reply by UncoveredPitch** [reply]
I have the same problem. None of the plug-ins I tried to load were recognised. I just gave up and reverted to beta5. I don't have the time to send any server logs because it takes a long time for a full scan—and I have to do it twice so I can get back to my mix which only runs with beta5. But it definitely looks like more than an isolated issue. My stats:

MacOS 10.14.6 Reaper 6.37
